### ISSUE TITLE
Landscape not handled anymore

### ISSUE THREAD

**Issue by kevin-hirsch** [AssistoLab/KVNProgress#81]
The problem is that in landscape the `KVNProgress` does not display itself correctly:

![bug](https://cloud.githubusercontent.com/assets/2794684/16771538/2677e3b6-4851-11e6-9141-61bc94bfa4e9.png)

The problem comes from the commit *Showing loader above keyboard* (a741fea)
I tested with the old code and it works perfectly:

```objc
- (void)addToView:(UIView *)superview
{
	UIWindow *currentWindow = [UIApplication sharedApplication].keyWindow;
	
	if (!currentWindow) {
		NSEnumerator *frontToBackWindows = [[[UIApplication sharedApplication] windows] reverseObjectEnumerator];
		
		for (UIWindow *window in frontToBackWindows) {
			if (window.windowLevel == UIWindowLevelNormal) {
				currentWindow = window;
				break;
			}
		}
	}
	
	if (self.superview != currentWindow) {
		[self addToView:currentWindow];
	}
}
```

To investigate...

**Comment by animaonline** [comment]
Hi, any updates on this one?

**Comment by aftabaig** [comment]
Any updates? I am creating a landscape only iPad app and need it urgently.

**Comment by kevin-hirsch** [comment]
Not for the moment, I will try fix it this week. Until, you can use commit b13f1cf78fcc98445875a95db872d7399b9205c9 just before a741fea that broke the landscape mode.
```ruby
pod 'KVNProgress', :git => 'https://github.com/AssistoLab/KVNProgress.git', :commit => 'b13f1cf78f'
```

**Comment by aftabaig** [comment]
Thanks Kevin.

**Comment by kevin-hirsch** [comment]
It's now fixed. It will be available in 2.4. Until then, feel free to use this version of KVNProgress:
```ruby
pod 'KVNProgress', :git => 'https://github.com/AssistoLab/KVNProgress.git', :commit => '302514b'
```

**Comment by kevin-hirsch** [comment]
Fix available in version **2.3.1**.
You can use it via `pod 'KVNProgress'` ✨
